README.md

```diff
@@ -1,13 +1,13 @@
 # Metis
 Metis is the service that annotates datasets with ontology terms using the Zooma application developed by the SPOT team at EBI. Metis is named after the ancient Greek Titaness of good counsel, advice, planning, cunning, craftiness, and wisdom [[1](http://greekmythology.wikia.com/wiki/Metis), [2](https://en.wikipedia.org/wiki/Metis_(mythology))].
-It takes a dataset and returns an updated dataset with ontology terms for 'assay'/'technology'/'tissue'/'disease' fields.
+It takes a field and term and responds with ontology terms for 'assay'/'technology'/'tissue'/'disease' fields.
 
 1. [http://greekmythology.wikia.com/wiki/Metis](http://greekmythology.wikia.com/wiki/Metis)
 2. [https://en.wikipedia.org/wiki/Metis_(mythology)](https://en.wikipedia.org/wiki/Metis_(mythology))
 
 ### Update Datasets with Ontology Terms
 
-Metis should use Iris to register a `...` pattern ? (TBD).
+Metis uses Iris to register a `action.annotate.metis` pattern.
```

Comment:
Let's not use the name of the service in the action. Just the action :slightly_smiling_face: